lib/rules/method.js

```diff
@@ -71,6 +71,21 @@ function checkCallExpression(ruleHelper, callExpr, node) {
         ruleHelper.checkMethod(callExpr);
         break;
 
+
+    case "SequenceExpression": {
+        // the return value of a SequenceExpression is the last expression.
+        // So, we create a new mock CallExpression with the actually called
+        // ... expression as the callee node and pass it to checkMethod()
+
+        /* eslint-disable no-case-declarations */
+        const newCallExpr = Object.assign({}, callExpr);
+        const idx = node.expressions.length - 1;
+        const called = node.expressions[idx];
+        newCallExpr.callee = called;
+        ruleHelper.checkMethod(newCallExpr);
+        break;
+        /* eslint-enable no-case-declarations */
```

Comment:
this isn't needed anymore.

Reply:
yeah...that happens when you try to fix a "simple thing" during meetings. patch coming up now.